src/commands/treeSitterMapping.ts

```diff
@@ -1,7 +1,18 @@
-import { binaryOperatorTranslations, ExpressionMetricMapping } from "../parser/helper/Model";
```

Comment:
Let's imrpove the name of this file (`treeSitterMapping.ts`). Suggestion: `UpdateTreeSitterGrammars` or `TreeSitterGrammarUpdater`.

Reply:
Mh, as it does not actually update the grammars themselves I have gone with "importNodeTypes", to make more clear that it imports the `node-types.json`-files and updates the `nodeTypesConfig.json`

Reply:
Alright. Though, I would like to agree on a consistent naming for all these things again later. There should be a documentation about what a node type or an expression is exactly. I will create an issue for that.